pkg/crc/network/types.go

```diff
@@ -12,3 +19,33 @@ type ResolvFileValues struct {
 	SearchDomains []SearchDomain
 	NameServers   []NameServer
 }
+
+type Mode string
+
+const (
+	DefaultMode Mode = "default"
```

Comment:
I do not see start options using these values, or the config help.
We need to somehow print possible options with help/description?